components/tor/tor_profile_service.cc

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/tor/tor_profile_service.h"
+
+#include <string>
+
+#include "brave/components/tor/pref_names.h"
+#include "brave/components/tor/tor_launcher_service_observer.h"
+#include "components/pref_registry/pref_registry_syncable.h"
+#include "components/prefs/pref_registry_simple.h"
+#include "components/prefs/pref_service.h"
+
+namespace tor {
+
+TorProfileService::TorProfileService() {
+}
+
+TorProfileService::~TorProfileService() {
```

Comment:
nit: = default

Reply:
addressed in e1726d757df0524a90d1c7a961847bebc7ed8eee